paddle/operators/conv_shift_op.cu

```diff
@@ -160,20 +160,20 @@ class ConvShiftGradKernel<platform::GPUPlace, T>
     auto stream = context.cuda_device_context().stream();
 
     const int x_per_block = 256;
-    int num_x_blocks = div_up(x_width, x_per_block);
+    int num_x_blocks = DivUp(x_width, x_per_block);
     dim3 grid_dim(num_x_blocks, y_width, batch_size);
 
     if (dX) {
       T *dx_data = dX->mutable_data<T>(context.GetPlace());
       cudaMemsetAsync(dx_data, 0, dX->numel() * sizeof(T), stream);
```

Comment:
You can clean memory in [this way](https://github.com/PaddlePaddle/Paddle/blob/2f3d1c345ff552678a2166e1e724e0af320cd427/paddle/operators/lstm_op.h#L200).